### DISCUSSION TITLE
Widgets on Android

### DISCUSSION THREAD

**Discussion by NicosNicolaou16** [community/community#78912] [Mobile] [answered]
It is a good idea to have widgets for contributions or other useful information for my profile.

**Comment by stevepopovich** [comment, marked as answer]
Hi @NicosNicolaou16 thanks for writing in. This is a great idea that we are considering internally. Ill note your feedback. Thank you!

**Reply by NicosNicolaou16** [reply]
Thank you very much! 😀
